.travis.yml

```diff
@@ -1,7 +1,7 @@
+sudo: false
 language: node_js
 node_js:
+  - 'stable'
```

Comment:
https://github.com/creationix/nvm/issues/870 something to think about. We might want to change `stable` to `node`. And at some point we'll be able to change `4.2` to `lts` or `argon` or something.